code_samples/README.md

```diff
@@ -0,0 +1,37 @@
+# Code Samples
+
+This document explains Hazelcast Node.js Client code samples which you can use to learn how to use Hazelcast features.
+
+**org-website/** — samples to display on https://hazelcast.org/
+
+**aggregation.js** — some built-in aggregation samples
+
+**custom_serializer.js** — creating and adding custom serializer to serialization config
+
+**distibuted_object_listener.js** — listening to distributed object events
+
+**flakeid_generator.js** — some basic flakeid generator operations
+
+**global_serializer.js** — creating and adding global serializer to serialization config
+
+**lifecycle_listener.js** — listening to lifecycle events of the Hazelcast instance
+
+**list.js** — some basic list operations
+
+**logging.js** — defining log levels
+
+**map.js** — some basic map operations
```

Comment:
I think `some basic` is not neccessary. We could go with `usage of distributed [map/flakeid/list] etc` where appropriate.